VSDiagnostics/VSDiagnostics/VSDiagnostics/VSDiagnosticsResources.resx

```diff
@@ -654,4 +654,10 @@
   <data name="TestMethodWithoutTestAttributeAnalyzerTitle" xml:space="preserve">
     <value>A method might be missing a test attribute.</value>
   </data>
+  <data name="GetHashCodeRefersToMutableFieldAnalyzerMessage" xml:space="preserve">
+    <value>GetHashCode() refers to {0}</value>
```

Comment:
This message does have a description of what exactly is wrong, except I build it into a string slightly differently than your suggestion. See https://github.com/VSDiagnostics/VSDiagnostics/pull/601/files#diff-42e7dce63789d70b5a21ed78a68cac2aR170